docs/lineage/airflow.md

```diff
@@ -1,18 +1,73 @@
 # Airflow Integration
 
-DataHub supports integration of 
+DataHub supports integration of
```

Comment:
Do you mind filling out the PR description to discuss what exactly is the scope of PR?

Reply:
Sure,  I will do